gimmik/kernels/c-omp.mako

```diff
@@ -1,14 +1,22 @@
 # -*- coding: utf-8 -*-
 
 void
-${funcn}(int ncol,
+% if n is None:
+${funcn}(int n,
          const ${dtype}* restrict b, int ldb,
          ${dtype}* restrict c, int ldc)
 {
+% else:
+${funcn}(const ${dtype}* restrict b, ${dtype}* restrict c)
```

Comment:
Is it worth getting gimmik to return a function header now that it can change? Or do you maybe want to bake this into the new API we have discussed?

Reply:
At the moment we should probably document that the signatures depend on the input arguments in a well defined way.